deployment/v3/mosip/resident/install.sh

```diff
@@ -12,29 +12,30 @@ CHART_VERSION=12.0.2
 echo Create $NS namespace
 kubectl create ns $NS
 
-echo Istio label 
+echo Istio label
 kubectl label ns $NS istio-injection=enabled --overwrite
 helm repo update
 
 echo Copy configmaps
 ./copy_cm.sh
 
-echo Installing Resident
-helm -n $NS install resident mosip/resident --version $CHART_VERSION
-
 API_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-api-internal-host})
-resident-ui_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-resident-ui-host})
+resident_ui_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-resident-ui-host})
+
+
+#echo Installing Resident
+helm -n $NS install resident mosip/resident --set istio.corsPolicy.allowOrigins.\[0\].prefix=$resident_ui_HOST --version $CHART_VERSION
 
 #echo Installing resident-ui-Proxy into Masterdata and Keymanager.
 #kubectl -n $NS apply -f resident-ui-proxy.yaml
 
 echo Installing resident-ui
-helm -n $NS install resident-ui mosip/resident-ui --set resident-ui.apiUrl=https://$API_HOST/v1/ --set istio.hosts\[0\]=$resident-ui_HOST --version $CHART_VERSION
+helm -n $NS install resident-ui mosip/resident-ui --set residentUi.apiHost=$API_HOST --set istio.hosts\[0\]=$resident_ui_HOST --version $CHART_VERSION
```

Comment:
Change residentUi.apiHost to resident.apiHost

Change $resident_ui_HOST to $RESIDENT_HOST